src/face-recognition/index.rst

```diff
@@ -37,27 +37,21 @@ Below we start DeepStack with only the face APIs enabled.
    -p 80:5000 deepquestai/deepstack:gpu
 
 
-*Basic Parameters*
-
-**-e VISION-FACE=True** This enables the face recognition APIs, all apis are disabled by default.
-
-**-v localstorage:/datastore** This specifies the local volume where deepstack will store all data.
-
-**-p 80:5000** This makes deepstack accessible via port 80 of the machine.
+Starting DeepStack on other Platforms
+-------------------------------------
 
+`DeepStack on Windows (CPU and GPU) <windows>`_
 
-Starting DeepStack on Raspberry PI
-----------------------------------
+`DeepStack on Raspberry PI <raspberry-pi>`_
 
-.. code-block:: bash
 
-   sudo deepstack start "VISION-FACE=True"
+*Basic Parameters*
 
+**-e VISION-FACE=True** This enables the face recognition APIs, all apis are disabled by default.
 
-Starting DeepStack on Windows
------------------------------
+**-v localstorage:/datastore** This specifies the local volume where deepstack will store all data.
```

Comment:
I am getting confused as there is a mix of deepstack and DeepsStack, I think its good to stick to one

Reply:
It's definitely **DeepStack**. It has been corrected.